book/en/src/by-example/resources.md

```diff
@@ -30,6 +30,11 @@ task.
 Thus, a task `#[local]` resource can only be accessed by one singular task.
 Attempting to assign the same `#[local]` resource to more than one task is a compile-time error.
 
+Types of `#[local]` resources must implement [`Send`] trait as they are being sent from `init`
```

Comment:
Is it appropriate to add the example of why this is reasonable? Here I'm referring to @japaric 's example with `Rc` type.

Reply:
It's fine without IMO.